src/node.cc

```diff
@@ -1653,6 +1654,21 @@ static void ReportException(Environment* env, const TryCatch& try_catch) {
 }
 
 
+void StartSigintWatchdog(const FunctionCallbackInfo<Value>& args) {
+  int ret = SigintWatchdogHelper::GetInstance()->Start();
+  if (ret != 0) {
+    Environment* env = Environment::GetCurrent(args);
+    env->ThrowErrnoException(ret, "pthread_create");
```

Comment:
I think the string here should not reference something platform-specific (pthreads is a POSIX thing).